lib/chef_zero/server.rb

```diff
@@ -164,10 +164,10 @@ def port
     #
     def url
       sch = @options[:ssl] ? 'https' : 'http'
-      @url ||= if @options[:host].include?(':')
-                 URI("#{sch}://[#{@options[:host]}]:#{port}").to_s
+      @url ||= if @options[:host].first.include?(':')
+                 URI("#{sch}://[#{@options[:host].first}]:#{port}").to_s
                else
-                 URI("#{sch}://#{@options[:host]}:#{port}").to_s
+                 URI("#{sch}://#{@options[:host].first}:#{port}").to_s
```

Comment:
Is there any middle ground? I guess I'm okay with it not working as long as it's somewhat easy to learn what the problem is should you run into it. Maybe return a 500 or something for sandbox creation when using a non-primary address?